fsr.ino

```diff
@@ -22,9 +24,15 @@
   }
   void ButtonPress(uint8_t button_num) {
     Joystick.button(button_num, 1);
+    #if defined(ENABLE_BUILTIN_LED_BLINK)
+      digitalWrite(LED_BUILTIN, HIGH);   // turn the LED on (HIGH is the voltage level)
```

Comment:
What should the LED be doing when you:

1. Press button 1
2. Press button 2
3. Release button 2 (but keep button 1 pressed).

With the current logic it would turn off the LED even though button 1 is still pressed. Is that your intention? I'm fine to keep it as is since it seems like it's just a nice and simple diagnostic tool for single arrows but was curious.